### PR TITLE
Add diamond mining drills, buff regular mining drills

### PR DESCRIPTION
**Pull request by Blackern5000**
## About the PR
Adds diamond mining drills, a T2 tech item which can be printed by science using diamonds. It is currently the only recipe to use diamonds. Also moves grappling tech to salvage T1, and makes diamonds worth 2k each instead of 500.
Also moves some prototypes around to better categorized files
- Pickaxes and drills are now found in mining.yml
- Drill recipes are now found in salvage.yml

## Why / Balance
Emo told me to do it
image

For awhile now, drills have been outclassed by roundstart pickaxes due to the fact that you can instantly break 3 rocks every wideswing, and it took drills 3 hits to break one single rock. I've buffed them to break rocks in 2 hits, and hit 4 times a second. Diamond drills are simply regular drills with doubled damage, so they break rocks in one hit and hit 4 times a second, easily outclassing pickaxes.

As for the research changes, Emo told me to do it.
image
It just makes more sense to lock higher level stuff behind higher level tech.

As for the diamond price change, Emo told me to do it in this pr instead of making it a separate one..

## Media

https://github.com/user-attachments/assets/23c0c821-44da-4159-be43-13ec2e467b44

image
## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Added diamond-tipped mining drills, a science printed item found under T2 industrial.
- tweak: Standard mining drills now attack faster and break rocks in two hits.
- tweak: Moved ore bags of holding to T2 industrial tech.
- tweak: Grappling guns are now researched with Salvage Equipment.
- tweak: Diamonds are now worth 2,000 spesos each.

### CODE REVIEW

**Comment by lzk228** [comment]
diamond pickaxe soon?

File: Resources/Prototypes/Reagents/Materials/materials.yml [hunk 1]
```diff
@@ -123,4 +123,12 @@
...
\ No newline at end of file
+  price: 0
+
+- type: material
+  id: Diamond
+  name: materials-diamond
+  unit: materials-unit-piece
+  icon: { sprite: Objects/Materials/materials.rsi, state: diamond }
+  color: "#80ffff"
+  price: 2.5 # This should probably be a lot more but I'll leave that to a different pr
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Reagents/Materials/materials.yml:134]:
```suggestion
  price: 10 
```
live dangerously, lad

**Review by EmoGarbage404 — APPROVED** [review]

**Comment by IProduceWidgets** [comment]
How would you feel about re-balancing the pickaxe to be a little slower and the shovel to be a little faster at mining?
Right now the pickaxe is fast enough there's not much reason to use a drill, and the shovel is so slow youd rather die than use it.

File: Resources/Prototypes/Reagents/Materials/materials.yml (hunk 1 above)
Comment (EmoGarbage404) [reply Resources/Prototypes/Reagents/Materials/materials.yml:134]: live dangerously

**Review by UbaserB — CHANGES_REQUESTED** [review]
The outline on the diamond drillhead should be a bit darker, the inside grooves are fine though.

**Review by UbaserB — APPROVED** [review]

**Pull request merged**